rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -820,8 +821,26 @@ private void emitRejectEvent(Coin value, String senderAddressStr, RejectedPegout
      */
     private boolean requestRelease(Address destinationAddress, Coin value, Transaction rskTx) throws IOException {
         if (activations.isActive(RSKIP219)) {
+            int pegoutSize = getRegularPegoutTxSize(getActiveFederation());
+            Coin feePerKB = getFeePerKb();
+            // The pegout transaction has a cost related to its size and the current feePerKB
+            // The actual cost cannot be asserted exactly so the calculation is approximated
+            // On top of this, the remainder after the fee should be enough for the user to be able to operate
+            // For this, the calculation includes an additional percentage to assert for this
+            Coin requireFundsForFee = feePerKB
+                .divide(1000) // Get the s/b
```

Comment:
You will ussually prefer to multiply and then divide

Reply:
1024 is a bit less confusing here, but it's just a design decision

Reply:
The 1000 comes from the interpretation of feePerKB bitcoinj-thin has. See here: https://github.com/rsksmart/bitcoinj-thin/blob/master/src/main/java/co/rsk/bitcoinj/wallet/Wallet.java#L728
I wanted to keep the same unit 😬